scripts/configs/PKGBUILD.template

```diff
@@ -28,6 +29,8 @@ package() {
   chown root:root -R "${pkgdir}/opt/zextras/<%- htmlWebpackPlugin.options.installMode %>/iris/${pkgname}/<%- htmlWebpackPlugin.options.commit %>"
   chmod 644 -R "${pkgdir}/opt/zextras/<%- htmlWebpackPlugin.options.installMode %>/iris/${pkgname}/<%- htmlWebpackPlugin.options.commit %>"
   find "${pkgdir}/opt/zextras/<%- htmlWebpackPlugin.options.installMode %>/iris/${pkgname}/<%- htmlWebpackPlugin.options.commit %>" -type d -exec chmod a+x "{}" \;
+  rm -rf "${pkgdir}/opt/zextras/<%- htmlWebpackPlugin.options.installMode %>/iris/${pkgname}/<%- htmlWebpackPlugin.options.commit %>/i18n"
```

Comment:
if you remove the copy in [webpack.build.config.js](https://github.com/zextras/carbonio-ui-sdk/pull/37/files#diff-e641c4b71893956d0e9611986db412d9382eead2c589e5ea7cb229f37cc967f9)

maybe you can remove the `rm -rf` command